### PR TITLE
Home card Collective description limited to 3 lines and expands on click

### PR DESCRIPTION
**Pull request by krisbitney**
Closes https://github.com/GoodDollar/GoodCollective/issues/31

### CODE REVIEW

File: packages/app/src/pages/HomePage.tsx [hunk 1]
```diff
@@ -27,27 +26,31 @@ function HomePage() {
...
-      try {
-        const t: any = [];
-        for (const e of requests) {
-          console.log(e.ipfs);
-          const res = await axios.get(`https://gateway.pinata.cloud/ipfs/${e.ipfs}`);
-          t.push({
+      const t: Promise<CollectiveHomeCardData>[] = [];
+      for (const e of requests) {
+        const promise: Promise<CollectiveHomeCardData> = axios
+          .get(`https://gateway.pinata.cloud/ipfs/${e.ipfs}`)
```
Comment (sirpy) [inline packages/app/src/pages/HomePage.tsx:32]: This should have been done via apollo+localstorage cache or other solution as was discussed with the original team.

**Review by sirpy — CHANGES_REQUESTED** [review]

File: packages/app/src/pages/HomePage.tsx (hunk 1 above)
Comment (krisbitney) [reply packages/app/src/pages/HomePage.tsx:32]: I was not with the original team. I cannot review all the prior communications you had with the original team, to the extent that records exist. 

I know you're unhappy with the service provided by the original team, and understandably so!

Please understand that I am here to help. I'm here to get this project back in shape.

To help me help you, please create a new GitHub issue for each distinct QA item you see.

This particular PR is about fixing the line limit in the collective descriptions, not about image caching. Please see https://github.com/GoodDollar/GoodCollective/issues/31

Comment (krisbitney) [reply packages/app/src/pages/HomePage.tsx:32]: Note that the requirements request that we avoid use of `localstorage`. It also requests that we avoid use of `window`. Has that requirement changed?

https://serota.notion.site/General-Technical-Requirements-e2b080782fbe4f75aa2125cef915fe9d

Comment (sirpy) [reply packages/app/src/pages/HomePage.tsx:32]: I ment AsyncStorage (on web its a wrapper around localstorage)

**Pull request merged**